Cargo.toml

```diff
@@ -251,7 +258,11 @@ tempfile = "3.1.0"
 filecheck = "0.5.0"
 libc = "0.2.60"
 file-per-thread-logger = "0.2.0"
-tokio = { version = "1.26.0" }
+tokio = { version = "1.26.0", features = [ "rt", "time" ] }
+hyper = "=1.0.0-rc.3"
+http = "0.2.9"
+http-body = "1.0.0-rc.2"
+http-body-util = "0.1.0-rc.2"
```

Comment:
While you're at it mind putting `=` constraints on these as well? (works around issues with Cargo's handling of pre-releases)